### PR TITLE
ada-url 2.2.0 (new formula)

### PR DESCRIPTION
**Pull request by anonrig**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/ada-url.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        auto url = ada::parse<ada::url_aggregator>("https://www.github.com/ada-url/ada");
+        url->set_protocol("http");
+        std::cout << url->get_protocol() << std::endl;
+        return EXIT_SUCCESS;
+      }
+    EOS
+
+    system ENV.cxx, "test.cpp", "-std=c++17",
+           "-I#{include}", "-L#{lib}", "-lada", "-o", "test"
+    assert_equal "\"Homebrew\"\n", shell_output("./test")
```
Comment (carlocab) [inline Formula/ada-url.rb:34]: Seems like this is what you should be expecting?
```suggestion
    assert_equal "http", shell_output("./test").chomp
```

File: Formula/ada-url.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        auto url = ada::parse<ada::url_aggregator>("https://www.github.com/ada-url/ada");
+        url->set_protocol("http");
+        std::cout << url->get_protocol() << std::endl;
+        return EXIT_SUCCESS;
+      }
+    EOS
+
+    system ENV.cxx, "test.cpp", "-std=c++17",
+           "-I#{include}", "-L#{lib}", "-lada", "-o", "test"
+    assert_equal "http", shell_output("./test").chomp
```
Comment (anonrig) [inline Formula/ada-url.rb:34]:
```suggestion
    assert_equal "http:", shell_output("./test").chomp
```

**Comment by anonrig** [comment]
Thank you @carlocab.

**Review by carlocab — DISMISSED** [review]
Thanks!

**Pull request merged**